docs/develop/topics/serializers.md

```diff
@@ -154,33 +154,41 @@ to (moving ``title`` to a top-level key and stripping ``akey``):
 
 ## Using schemas
 
-The schema can the be combined with the Marshmallow JSON serializer to
-output JSON:
+The schema can the be combined with the Marshmallow serializer, it takes three arguments:
```

Comment:
Small typo here I think
`The schema can be combined with the Marshmallow serializer, it takes three arguments:`
or maybe
`The schema can then be combined with the Marshmallow serializer, it takes three arguments:`